app/src/main/res/layout-sw600dp-port/question_player_fragment.xml

```diff
@@ -95,7 +95,34 @@
         android:progressDrawable="@drawable/progress_bar"
         app:layout_constraintBottom_toTopOf="@id/question_progress_text"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent" />
+        app:layout_constraintStart_toEndOf="@id/hints_and_solution_fragment_container" />
```

Comment:
Update this to `app:layout_constraintStart_toStartOf="parent" />`
 because marginStart=64 and marginEnd=64dp makes sure that the progressbar never overlaps with the `hints_and_solution_fragment_container `

So technically we don't need any change in this file except that of `android:layout_width="0dp"` in ProgressBar

Reply:
done

Reply:
Ditto as above.